dev.bsnes.bsnes.yaml

```diff
@@ -0,0 +1,26 @@
+app-id: dev.bsnes.bsnes
+runtime: org.gnome.Platform
```

Comment:
Do you really need the GNOME runtime? Isn't the freedesktop SDK enough?

Reply:
Indeed, the freedesktop runtime is enough :+1: